src/buildHTML.js

```diff
@@ -376,6 +376,7 @@ groupTypes.supsub = function(group, options, prev) {
         supsub.children[1].style.marginRight = scriptspace;
     }
 
+    supsub.classes.push("msupsub");
```

Comment:
Fixed, maybe I should've inlined this call in the final final makeSpan call.  Just let me know.